### DISCUSSION TITLE
v2.8.13

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4834] [分享 / Share]
<details>
<summary>获得 v2.8.12-preview 的所有新功能, 点击展开查看</summary>

✨新增
- `下载视频 - WASM 混流输出` 支持批量下载. (PR #4726 by [WakelessSloth56](https://github.com/WakelessSloth56))
- `夜间模式` 增加 `color-scheme` meta 信息, 适配 Chrome 的 [Fluent overlay scrollbars](chrome://flags/#fluent-overlay-scrollbars). (#4717)

</details>

✨新增
- `删除广告` 支持屏蔽首页的 "广告已被屏蔽" 占位卡片.
- `自定义顶栏` 增加公益、公开课、社区中心入口. (#1511)

🐛修复
- 修复 `显示视频投稿时间` 时间显示错误及不显示时间的问题. (#4755, PR #4756 by [呼乎户](https://github.com/wisokey))
- 修复夜间模式的一些适配问题. (#4376, #4768, #4785, #4802, PR #4763, #4810 by [Pencil](https://github.com/pencilqaq))
- 修复 `屏蔽黑名单up主` 弹窗高度溢出. (PR #4784 by [snowraincloud](https://github.com/snowraincloud))
- 修复 `自定义顶栏` 搜索框在 b 站搜索页打字时自动弹出. (#4808)

☕开发者相关
- 修复 GitHub Workflow 中 pnpm 安装失败.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.13'>v2.8.13</a>.</em>

**Comment by the1812** [comment]
你可能还想问[评论区那些问题](https://github.com/the1812/Bilibili-Evolved/discussions/4804)修了吗, 答案是没有. 这次只是把这两个月的存货发一下. (好不容易过段安稳日子 b 站又开始搞事了)
另外还有个糟糕的消息, 视频的弹幕也改版了, 所以弹幕相关的功能也会失效, 下个版本预计主要处理这两块失效的问题.

**Reply by Tinhone** [reply]
想问下评论区功能和弹幕功能失效的问题还会再开一个集中跟进的 issue 嘛

**Reply by the1812** [reply]
会

**Reply by the1812** [reply]
#4843
